### PR TITLE
MMCA-4472-update to play 2.9

### PR DESCRIPTION
**Pull request by AdamCoops**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -46,13 +48,12 @@ lazy val microservice = Project(appName, file("."))
...
-    Test / parallelExecution         := false,
-    Test / fork                      := false
+    scalaVersion := scala2_13_12,
+    targetJvm := "jvm-11",
+    scalacOptions := Seq("-feature", "-deprecation"),
+    Test / parallelExecution := false,
+    Test / fork := false
   )
-  .configs(IntegrationTest)
   .settings(addTestReportOption(IntegrationTest, "int-test-reports"))
```
Comment (tyagikumarvineet) [inline build.sbt]: Remove this line as it is not required after these updates

File: build.sbt [hunk 2]
```diff
@@ -46,13 +48,12 @@ lazy val microservice = Project(appName, file("."))
       compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
       "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
     ),
-    scalaVersion                     := scala2_13_8,
-    targetJvm                        := "jvm-11",
-    scalacOptions                    := Seq("-feature", "-deprecation"),
-    Test / parallelExecution         := false,
-    Test / fork                      := false
+    scalaVersion := scala2_13_12,
```
Comment (tyagikumarvineet) [inline build.sbt]: This is redundant as we have already declared it at line 16 so please remove it

File: conf/prod.routes [hunk 3]
```diff
@@ -1,5 +1,3 @@
 # Add all the application routes to the app.routes file
 ->         /customs-financials-email-throttler              app.Routes
 ->         /                          health.Routes
```
Comment (tyagikumarvineet) [inline conf/prod.routes]: Could you please health.Routes with app.Routes? I know, its not part of your changes but would be good to align this. If you don't want then it is still okay as it should have been done as part of Live services ticket

File: test/uk/gov/hmrc/customs/financials/emailthrottler/services/MetricsReporterServiceSpec.scala [hunk 4]
```diff
@@ -34,45 +33,61 @@ class MetricsReporterServiceSpec extends SpecBase {
...
-        verify(mockRegistry)
-          .counter(
-            ArgumentMatchers.eq("email-queue.mark-oldest-send-email-job-for-processing-in-mongo-successful"))
+
+        verify(mockRegistry).counter(ArgumentMatchers.eq(
+          "email-queue.mark-oldest-send-email-job-for-processing-in-mongo-successful"))
+
         verify(mockCounter).inc()
       }
+
```
Comment (tyagikumarvineet) [inline test/uk/gov/hmrc/customs/financials/emailthrottler/services/MetricsReporterServiceSpec.scala:60]: Remove this extra line break

File: project/AppDependencies.scala [hunk 5]
```diff
@@ -2,18 +2,18 @@ import sbt.*
...
+    "uk.gov.hmrc" %% "bootstrap-backend-play-29" % bootstrapVersion
   )
 
   val test: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc" %% "bootstrap-test-play-28" % bootstrapVersion % Test,
+    "uk.gov.hmrc" %% "bootstrap-test-play-29" % bootstrapVersion % Test,
     "org.pegdown" % "pegdown" % "1.6.0" % "test",
-    "org.scalatestplus.play" %% "scalatestplus-play" % "5.0.0" % "test",
+    "org.scalatestplus.play" %% "scalatestplus-play" % "7.0.1" % "test",
     "org.mockito" % "mockito-core" % "3.1.0" % "test"
```
Comment (tyagikumarvineet) [inline project/AppDependencies.scala]: Please update this lib to 5.10.0

**Review by harshare — APPROVED** [review]
👍

**Review by tyagikumarvineet — APPROVED** [review]
Test coverage (to make it to min coverage) for Branch will be covered as part of separate ticket.

Also the tests written for SendEmailJob.scala will be amended as part of that ticket

**Pull request merged**